packages/RichText-Tests.package/RTEHeaderComponentTest.class/instance/testH1LargerH2LargerH3.st

```diff
@@ -1,18 +1,15 @@
 testing
 testH1LargerH2LargerH3
 
-	| h1size h2size h3size |
-	self click: (self menuItemNamed:  'h1').
-	h1size := self currentFontSize.
+	| currentFontSizes |
 	
-	self click: (self menuItemNamed: 'h2').
-	h2size := self currentFontSize.
+	currentFontSizes := Array new: 3.
+	1 to: 3 do: [:i|
+		self click: (self menuItemNamed: 'h', i).
+		currentFontSizes at: i put: self currentFontSize].
 	
-	self click: (self menuItemNamed: 'h3').
-	h3size := self currentFontSize.
-	
-	self assert: h1size > h2size.
-	self assert: h2size > h3size. 
+	self assert: (currentFontSizes at: 1) > (currentFontSizes at: 2).
```

Comment:
idea: move the assert into the loop and test if the previous size is larger than the currentFontSize. Edge case would be the first element. If you have an idea for that I would prefer to change it because it's easier to extent. (but obviously not needed)